cmd/pipehub/main.go

```diff
@@ -135,3 +143,31 @@ func cmdGenerateRun(configPath, workspacePath *string) func(*cobra.Command, []st
 		}
 	}
 }
+
+func cmdVersion() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "version",
+		Short: "Version the application",
+		Long:  `Version the application server.`,
+		Run:   cmdVersionRun(gitCommit, goVersion, built, version),
+	}
+
+	cmd.Flags().StringVarP(&version, "version", "", "", "show version")
+	return &cmd
+}
+
+func cmdVersionRun(gitCommit, goVersion, built, version string) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
+
+		// Observe that the third line has no trailing tab,
+		// so its final cell is not part of an aligned column.
+		const padding = 3
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, padding, ' ', tabwriter.TabIndent)
+		fmt.Fprintln(w, "PipeHub:")
+		fmt.Fprintln(w, "    Version:\t", version)
```

Comment:
A good thing here is on each `fmt.Fprintln` check if the value is empty or not, if it is, we could skip the whole line. What do you think of it?

Reply:
fixed :D